tool/common/common.go

```diff
@@ -64,7 +64,7 @@ func (e *SessionsCollection) WriteText(w io.Writer) error {
 			id = session.GetSessionID()
 			typ = session.Protocol
 			participants = strings.Join(session.Participants, ", ")
-			hostname = session.ServerAddr
+			hostname = session.ServerHostname
```

Comment:
this could be a k8s session too. perhaps check the protocol or `ssh` or `kube` on which field to use.

Reply:
I took a similar approach to what we did in #44496 for `tsh sessions ls`: renamed "hostname" to target and used the kube cluster as the target for kube sessions.

Reply:
thanks for clarifying!